README.md

```diff
@@ -0,0 +1,16 @@
+# Pyxmex
+Parse American Express EPTRN files in python
+
+## Installation
+`$ pip install pyxmex`
+
+## Usage
+```python
+from pyxmex import Parser
+parser = Parser()
+
+parsed = parser.parse_detail_record_from_file('MYCOMPANY.EPTRN')
```

Comment:
this function name seems a little verbose. i feel the ideas of detail_record and from file should not be explicit in the function name.